src/target/riscv/riscv-013.c

```diff
@@ -928,6 +927,51 @@ static int dm013_select_target(struct target *target)
 	return dm013_select_hart(target, info->index);
 }
 
+#define EXECUTE_ABSTRACT_COMMAND_BATCH_SIZE 2
```

Comment:
This could be called just:

```suggestion
#define ABSTRACT_COMMAND_BATCH_SIZE 2
```

Reply:
Addressed.